ox-typst.el

```diff
@@ -34,8 +34,8 @@
 
 ;; Variables
 
-(defcustom org-typst-bin "typst"
-  "Path or name of Typst binary."
+(defcustom org-typst-process "typst c \"%s\""
+  "Format string for the command to process a Typst file to a PDF file."
```

Comment:
```suggestion
(defcustom org-typst-process "typst c \"%s\""
  "Format string for the command to process a Typst file to a PDF file.

The string is formatted with the file path of the Typst file.  The resulting
string must be a valid shell command, including a program name or a valid file
path to a Typst binary.  It is recommended to put the file path in quotes,
like the default value does.

Check the documentation of your Typst version for supported arguments.
"
```

I guess there is some additional information missing here. To prevent confusion or mistakes I would like to add this additional text. If you think something is missing, then feel free to extend it.